frontend/src/components/common/form/Check.tsx

```diff
@@ -85,7 +85,6 @@ export const Check: React.FC<CheckProps> = ({
             required={required}
             disabled={disabled}
             custom={custom}
-            defaultChecked={!!checked}
```

Comment:
Having both checked and defaultchecked throws a warning, controlled components should be using the checked prop so I've switched to that.